.github/CONTRIBUTING.md

```diff
@@ -34,17 +34,28 @@ actual development.
  * Pull requests should be atomic; Make one commit for each distinct change, so if a part
  of a pull request needs to be removed/changed, you may simply modify that single commit.
  Due to limitations of the engine, this may not always be possible; but do try your best.
- * Document and explain your pull requests thoroughly. Detail what each commit changes,
- and why it changes it. We do not want to have to read all of you commit names to figure
- out what your pull request is about.
+
+ * Document and explain your pull requests thoroughly. Failure to do so will delay a PR as
+  we question why changes were made. This is especially important if you're porting a PR 
+  from another codebase (i.e. TG) and divert from the original. Explaining with single
+  comment on why you've made changes will help us review the PR faster and understand your
+  decision making process.
+
  * Any pull request must have a changelog, this is to allow us to know when a PR is deployed 
  on the live server. Inline changelogs are supported through the format described 
  [here](https://github.com/ParadiseSS13/Paradise/pull/3291#issuecomment-172950466)
  and should be used rather than manually edited .yml file changelogs.
+
  * Pull requests should not have any merge commits except in the case of fixing merge
  conflicts for an existing pull request. New pull requests should not have any merge
  commits. Use `git rebase` or `git reset` to update your branches, not `git pull`.
 
+ * Please explain why you are submitting the pull request, and how you think your change will be beneficial to the game. Failure to do so will be grounds for rejecting the PR.
```

Comment:
perhaps we can add a section to the PR template "why it's good for the game" like TG has it

Reply:
Might think about it in like, 10 hours. I'm already supposed to sleep 2 hours ago and good ideas keep popping up just as I should be sleeping.